lts.json

```diff
@@ -21,21 +21,21 @@
   "ember-cli-sri": "^2.1.0",
   "ember-cli-test-loader": "^1.1.0",
   "ember-cli-uglify": "^1.2.0",
-  "ember-computed-decorators": "~0.2.0",
-  "ember-concurrency": "~0.7.0",
+  "ember-computed-decorators": "~0.2.2",
+  "ember-concurrency": "~0.7.15",
   "ember-css-modules": "~0.5.0",
```

Comment:
Actually, I'm changing my mind on this - we can introduce this without using it, correct?  Everyone's prior SASS will still work with the right build settings